config/routes.rb

```diff
@@ -11,4 +11,8 @@
   resources :vendors, only: [:index, :show]
 
   root to: "stats#index"
+
+  scope "v1" do
+    # NOTE: API controller routes go in this block
+  end
```

Comment:
so our experience with prescat was that ... we can never change the version of the API because it's too cumbersome -- how do you have openapi for multiple versions?  How many consumers are there, really, and is it worth it?   And probably other reasons I don't remember from a few discussions with @jmartin-sul.  

How many consumers will this app have?  How will versioning be used?   It may not be worth it.

Reply:
@ndushay It does feel a bit YAGNI given we've never incremented an API version before within SDR. Currently, most of our API apps *do* use a scope to wrap API operations though, e.g.:

* DSA: `/v1`
* DIA: `/dor`
* prescat: `/v1`
* sdr-api: `/v1`
* techmd: `/v1`

I'm thinking that using `/v1` for consistency, weighed against what you wrote, is of limited value. But particular in an app that is part UI and part API, I do think it's helpful to reflect that certain URLs in the app are API-focused. (I probably couldn't defend that too strongly.)

Long story short: how about a scope of `/api` instead of `/v1`?

Reply:
I made this change.